lib/repl.js

```diff
@@ -1548,23 +1548,20 @@ function defineDefaultCommands(repl) {
     help: 'Load JS from a file into the REPL session',
     action: function(file) {
       try {
-        var stats = fs.statSync(file);
+        const stats = fs.statSync(file);
         if (stats && stats.isFile()) {
           _turnOnEditorMode(this);
-          var data = fs.readFileSync(file, 'utf8');
-          var lines = data.split('\n');
-          for (var n = 0; n < lines.length; n++) {
-            if (lines[n])
-              this.write(`${lines[n]}\n`);
```

Comment:
The changed behavior here is also a bugfix: in case the saved load would be a template string with multiple line breaks, it would have changed the users data. That is not good and we should always save all data 1-to-1.